mockserver.js

```diff
@@ -131,9 +131,14 @@ var mockserver = {
         }
 
         var watchedHeaders = module.exports.headers;
+
         if(watchedHeaders && !Array.isArray(watchedHeaders)) {
             watchedHeaders = [watchedHeaders];
         }
+        if(!watchedHeaders || !watchedHeaders.length) {
+            watchedHeaders = ['authorization']
+        }
```

Comment:
I'm not sure lots of people will actually need to use authorization by default -- what's the use case rather than just doing `mockserver.headers = ['authorization']`?